collections/guides/setup_first_reactor/setup_first_reactor.md

```diff
@@ -62,11 +62,15 @@ The following example is for an [Atomic Card](https://docs.basistheory.com/api-r
 </span>
 
 ```js
-curl "https://api.basistheory.com/atomic/cards/<atomic_card_id>/react" \
+curl "https://api.basistheory.com/reactors/<reactor_id>/react" \
   -H "BT-API-KEY: <application_api_key>" \
   -X "POST" \
   -d '{
-    "reactor_id": "<reactor_id>"
+    "args": {
+      "card": {
+        "number": "{{card_number_token_id}}"
```

Comment:
```suggestion
        "number": "{{ <card_number_token_id> }}"
```

Reply:
`{{` are part of our syntax, and `<` represent something the user should replace before calling the API.